dali_mqtt_daemon.py

```diff
@@ -375,6 +376,56 @@ def on_message_brightness_cmd(mqtt_client, data_object, msg):
     except KeyError:
         logger.error("Lamp %s doesn't exists", light)
 
+def on_message_brightness_get_cmd(mqtt_client, data_object, msg):
+    """Callback on MQTT brightness get command message."""
+    logger.debug("Brightness Get Command on %s: %s", msg.topic, msg.payload)
+    light = re.search(
+        MQTT_BRIGHTNESS_GET_COMMAND_TOPIC.format(data_object["base_topic"], "(.+?)"),
+        msg.topic,
+    ).group(1)
+    try:
+        if 'group_' in light:
+          """ Check if the comand is for a dali group """
+          group = int(re.search('group_(\d+)', light).group(1))
+          lamp_object=data_object["all_lamps"][group]
+        else:
+          """ The command is for a single lamp """
+          if light not in data_object["all_lamps"]:
+              raise KeyError
+          lamp_object = data_object["all_lamps"][light]
+          
+        level = None
+        try:
+            level = data_object["driver"].send(gear.QueryActualLevel(lamp_object.short_address))
+
+            logger.debug("Get light <%s> results in %d", light, level.value)
+
+            mqtt_client.publish(
+                MQTT_BRIGHTNESS_STATE_TOPIC.format(data_object["base_topic"], light),
+                level.value,
+                retain=False,
+            )
+            if level.value == 0:
+              lamp_state=MQTT_PAYLOAD_OFF
+            else:
+              lamp_state=MQTT_PAYLOAD_ON
+                
+            mqtt_client.publish(
+                MQTT_STATE_TOPIC.format(data_object["base_topic"], light),
+                lamp_state,
```

Comment:
Can be made inline:

`MQTT_PAYLOAD_ON if level.value != 0 else MQTT_PAYLOAD_OFF`